docs/workflows/index.md

```diff
@@ -4,6 +4,7 @@
 
 ## Code-based workflows
 
-- [Create an Airtable integration with code](airtable-with-code.md)
-- [Create a Google Sheets integration with code](google-sheets-with-code.md)
-- [Create a Zip Code integration with code contract](zip-code-with-contract.md)
+- [Create an Airtable integration](airtable.md)
+- [Create a Google Sheets integration](google-sheets.md)
+- [Create a REST API integration](rest-api.md)
+- [Create a REST API integration with code](rest-api-with-code.md)
```

Comment:
Should the top three bullet points be under the "UI-based workflows" header? It's a bit confusing to see "Create a REST API integration" and "Create a REST API integration with code" together here.

Reply:
Also "with code" / "Code-based" might be the wrong nomenclature since they both utilize code, it's more about the starting point. Maybe "UI-configured sources" versus "Code-generated sources" or something like that?

Reply:
Addressed in https://github.com/Automattic/remote-data-blocks/pull/243/commits/2fb45d6ab5007529743762b2cc42a8cca43c1e5a